src/main/java/com/spring/onlinebookstore/model/CartItem.java

```diff
@@ -0,0 +1,35 @@
+package com.spring.onlinebookstore.model;
+
+import jakarta.persistence.Column;
+import jakarta.persistence.Entity;
+import jakarta.persistence.FetchType;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import jakarta.persistence.JoinColumn;
+import jakarta.persistence.ManyToOne;
+import jakarta.persistence.Table;
+import lombok.Getter;
+import lombok.Setter;
+import org.hibernate.annotations.Fetch;
+import org.hibernate.annotations.FetchMode;
+
+@Entity
+@Table(name = "cart_items")
+@Getter
+@Setter
+public class CartItem {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+    @ManyToOne(fetch = FetchType.LAZY, optional = false)
+    @JoinColumn(name = "shopping_cart_id")
+    @Fetch(FetchMode.JOIN)
+    private ShoppingCart shoppingCart;
```

Comment:
Please, check do you always need to load this relation, because FetchMode.JOIN loads them eagerly. It means you will always have a select query with `join`, which can have a huge impact on performance.
You can use @Query or @EntityGraph annotations in your repositories instead